packages/create-snowpack-app/index.js

```diff
@@ -102,7 +102,9 @@ const {
   targetDirectoryRelative,
   targetDirectory,
 } = validateArgs(process.argv);
-const installedTemplate = path.join(targetDirectory, "node_modules", template);
+const installedTemplate = template.startsWith(".")
+  ? path.resolve(targetDirectory, template) // handle local template
+  : path.join(targetDirectory, "node_modules", template); // handle template from npm/yarn
```

Comment:
This is a potentially-controversial change. It allows `create-snowpack-app` to use local templates, not just ones published to npm/GitHub. Feature, possibly?